jwst/wfs_combine/wfs_combine.py

```diff
@@ -119,13 +119,13 @@ def do_all(self):
         # Create a new model using the combined arrays...
         new_model = datamodels.ImageModel(data=data_c, dq=dq_c, err=err_c)
         new_model.update(self.input_1)
-        new_model.history.append(f"Flip dithers = {self.flip_dithers}")
```

Comment:
Does the history go to FITS header somewhere? Should we do both to preserve backward compatibility or it doesn't matter?